authentication_service/helpers/account/password_helpers.go

```
@@ -0,0 +1,48 @@
+package account
+
+import (
+	"fmt"
+	"log"
+	"io"
+
+	"golang.org/x/crypto/bcrypt"
+	"crypto/rand"
+)
+
+var INTEGER_TABLE = [...]byte{'1', '2', '3', '4', '5', '6', '7', '8', '9', '0'}
+
+
+//EncodeToString is used to create a 6 digit verification code 
+func EncodeToString(max int) string {
+    b := make([]byte, max)
+    n, err := io.ReadAtLeast(rand.Reader, b, max)
+    if n != max {
+        panic(err)
```

would it be better to return the error up rather than `panic`-ing?